lms/services/canvas_api.py

```diff
@@ -33,202 +48,351 @@ def get_token(self, authorization_code):
         """
         Get an access token for the current LTI user.
 
-        Posts to the Canvas API to get the access token and returns it.
```

Comment:
There were lots of comments saying we posted to the Canvas API, or made HTTP requests or soemthing similar. This seems obvious from the context